iamidentityv1/iam_identity_v1.go

```diff
@@ -42,7 +42,7 @@ type IamIdentityV1 struct {
 }
 
 // DefaultServiceURL is the default URL to make service requests to.
-const DefaultServiceURL = "https://iam.cloud.ibm.com"
+const DefaultServiceURL = "https://iam.test.cloud.ibm.com"
```

Comment:
@padamstx Is this service URL okay? Since the API definition is from the `production` branch.

Reply:
No, which API definition did you re-gen the code from?   I see that the production branch version of the API def uses the production URL.